### DISCUSSION TITLE
Managed variants separate for each institute?

### DISCUSSION THREAD

**Discussion by sofievonlanthen** [Clinical-Genomics/scout#4422] [Ideas]
Managed variants highlighted on the case page is not institute specific, meaning that variants classified as managed variants by other institutes are flagged for everyone (as far as I understand). 

Should these be institute specific, similar to matching causatives?

**Comment by dnil** [comment]
They could become! The initial discussion had it as beneficial to have a common knowledge pool, but if we agree, they can be pulled apart.

**Comment by dnil** [comment]
We have a Scout user meeting next week; could be a discussion topic for then!

**Comment by sofievonlanthen** [comment]
Sounds good. Had completely missed the Scout user meeting for next week.
